bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -119,15 +119,23 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
         debug_assert!(found);
 
         let caller_balance = sender_account.balance;
-        let insufficient_balance = call.value > caller_balance;
+        let is_call_or_callcode = call.kind == CallKind::Call || call.kind == CallKind::CallCode;
+        let insufficient_balance = call.value > caller_balance && is_call_or_callcode;
+
+        log::info!(
+            "insufficient_balance: {}, call type: {:?}, sender_account: {:?} ",
+            insufficient_balance,
+            call.kind,
+            call.caller_address
+        );
```

Comment:
don't need this high level logging. `insufficient_balance` is a "normal exec state", don't need to be highlight as log::info. debug and trace will be ok.

Reply:
updated in [b5f92fb](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1069/commits/b5f92fb6dc693ba2e826ae879e46b61252a5dd8b) thanks !!